### DISCUSSION TITLE
Suddenly unable to connect to wifi

### DISCUSSION THREAD

**Discussion by celticsamurai** [OnionUI/Onion#1624] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.1-1

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

All of a sudden, last week, my Miyoo Mini Plus has no longer been able to connect to our home Wi-Fi network. No updates happened prior to this. I was actively using this with FileZilla just so that I don't have to plug and unplug the micro SD card. I've since updated OnionOS and even my firmware.

### Steps to Reproduce

Go to settings > wifi > enter password > fail to connect

### Provide any additional context or information that may help us investigate your issue

I've attempted to split my wifi to hopefully connect to the 2.4ghz version, which has also failed. I've also tried changing the security settings, all of which made no difference.

**Comment by hotgar** [comment]
Try connecting via WPS if that's possible. [My Wi-Fi won't connect as it contains special characters, what do I do?](https://onionui.github.io/docs/faq#my-wi-fi-wont-connect-as-it-contains-special-characters-what-do-i-do). 
Also in case you have a mikrotik router i could share the configuration i stumbled upon that seems to be working.
